src/impl/util.js

```diff
@@ -197,7 +197,7 @@ export function normalizeObject(obj, normalizer, ignoreUnknown = false) {
       if (v !== null && !isUndefined(v) && !Number.isNaN(v)) {
         const mapped = normalizer(u, ignoreUnknown);
         if (mapped) {
-          normalized[mapped] = v;
+          normalized[mapped] = Number(v);
```

Comment:
This isn't quite right, because it lets NaN into this. You need to convert to a Number above that `if` check a few lines above.

Reply:
Thanks, @icambron for the review.